versions/3.1.1.md

```diff
@@ -357,7 +377,7 @@ Field Name | Type | Description
 <a name="infoTermsOfService"></a>termsOfService | `string` | A URI for the Terms of Service for the API. This MUST be in the form of a URI.
 <a name="infoContact"></a>contact | [Contact Object](#contactObject) | The contact information for the exposed API.
 <a name="infoLicense"></a>license | [License Object](#licenseObject) | The license information for the exposed API.
-<a name="infoVersion"></a>version | `string` | **REQUIRED**. The version of the OpenAPI document (which is distinct from the [OpenAPI Specification version](#oasVersion) or the API implementation version).
+<a name="infoVersion"></a>version | `string` | **REQUIRED**. The version of the OpenAPI document (which is distinct from the [OpenAPI Specification version](#oasVersion) or the version of API being described).
```

Comment:
```suggestion
<a name="infoVersion"></a>version | `string` | **REQUIRED**. The version of the OpenAPI document (which is distinct from the [OpenAPI Specification version](#oasVersion) or the version of the API being described).
```

Reply:
@ralfhandl can you move this over to your typo PR so it goes into 3.0.4 as well?  Then we can port that PR.  After how confusing it was for me to backport merges with that discriminator PR, I'd rather avoid that here and keep everything moving in one direction: 3.0.4 -> 3.1.1 -> 3.2.0

Reply:
Or I guess if the 3.04 PR has every change found here in it, we could incorporate those changes here and not merge forward?   (sorry, I should have atleast one cup of tea before commenting on PRs... thinking this through...)

Reply:
No worries, let's merge this monster as is (after a re-approval by @lornajane or @miqui) and then merge & forward-port #3924 separately.

Reply:
Thanks @ralfhandl - I have a little script that can merge a specified commit across brancheslfiles, so I can run that once your PR is merged if you'd like.

Reply:
Yes, please!

Reply:
@ralfhandl I have the PR porting the remaining 3.0.4 things ready to go as soon as this PR is merged.